stack/C/stackUsingArray.c

```diff
@@ -0,0 +1,32 @@
+#include<stdio.h>
+#include<stdlib.h>
+void main() {
+	void push(int *arr, int *top, int n, int i);
+	int pop(int *arr, int *top);
+	// number of elements
+	int n = 6;
+	int arr[n], top = -1;
+	// push all elements into the stack
+	for(int i = 0; i < n; i++) {
+		push(arr, &top, n, i);
+	}
+	printf("\n");
+	// pop elements one by one.
+	for(int i = 0; i < n; i++) {
+		printf("%d ", pop(arr, &top));
+	}
+}
+void push(int *arr, int *top, int n, int i) {
+	if(*top == n)
+		printf("\nStack is full. \n");
```

Comment:
An else should appear on the same line as the preceding }  [whitespace/newline] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*